openpype/hosts/houdini/plugins/publish/validate_mesh_is_static.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""Validator for correct naming of Static Meshes."""
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from openpype.pipeline.publish import ValidateContentsOrder
+
+from openpype.hosts.houdini.api.action import SelectInvalidAction
+from openpype.hosts.houdini.api.lib import get_output_children
+
+import hou
+
+
+class ValidateMeshIsStatic(pyblish.api.InstancePlugin,
+                           OptionalPyblishPluginMixin):
+    """Validate mesh is static.
+
+    It checks if output node is time dependant.
+    """
+
+    families = ["staticMesh"]
+    hosts = ["houdini"]
+    label = "Validate Mesh is Static"
+    order = ValidateContentsOrder + 0.1
+    actions = [SelectInvalidAction]
+
+    def process(self, instance):
+
+        invalid = self.get_invalid(instance)
+        if invalid:
+            nodes = [n.path() for n in invalid if isinstance(n, hou.Node)]
```

Comment:
What other types than `hou.Node` are you expecting? And if `get_invalid` is returning those are we sure we want them returned to the `SelectInvalidAction`?

Reply:
I think it's redundant, I will test it again.